toolkit/tools/pkg/imagecustomizerlib/cleanUpImage.go

```diff
@@ -0,0 +1,37 @@
+package imagecustomizerlib
+
+import (
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/safechroot"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/shell"
+	"github.com/sirupsen/logrus"
+)
+
+func cleanTdnfCache(imageChroot *safechroot.Chroot) error {
+
+	logger.Log.Infof("Cleaning up image")
+
+	// clear tdnf cache
+	tdnfClean := func() error {
+		tdnfArgs := []string{
+			"-v", "clean", "all",
+		}
+		stdoutCallback := func(line string) {
```

Comment:
Just use `LogLevel(logrus.TraceLevel, logrus.DebugLevel)`, You don't need to use `StdoutCallback` unless you want to do custom filtering.

Reply:
Updated